MailCore/Utils/Error+Extension.swift

```diff
@@ -42,7 +42,9 @@ private func displayErrorIfNeeded(error: Error) {
     @InjectService var snackbarPresenter: SnackBarPresentable
     if let error = error as? MailError {
         if error.shouldDisplay {
-            snackbarPresenter.show(message: error.errorDescription)
+            if let errorDescription = error.errorDescription {
```

Comment:
Maybe unwrap in same if so that we can send it to Sentry just in case ?

Reply:
I wondered if I should do it! I'll do this 👍